main.js

```diff
@@ -0,0 +1,11 @@
+const navBar = document.querySelector('#desktop-nav');
+const hamburguerButton = document.querySelector('.menu-bars');
+const hamburguerIcon = document.querySelector('.navbar-icon');
+
+function toggleMobileMenu() {
+  hamburguerIcon.classList.toggle('fa-bars');
+  hamburguerIcon.classList.toggle('fa-times');
+  navBar.classList.toggle('show-me');
```

Comment:
- [ ] As the last reviewer kindly suggest, the mobile menu should disappear after a click on one of the options. 

Please fix that as it will improve the view of your project. 😉